Sources/_TestingInternals/Discovery.cpp

```diff
@@ -86,6 +86,23 @@ struct SWTRelativePointer {
   }
 };
 
+/// A type representing a 32-bit absolute function pointer, usually used on platforms
+/// where relative function pointers are not supported.
+///
+/// This type is derived from `AbsoluteFunctionPointer` in the Swift repository.
+template <typename T, int32_t maskValue = 0>
```

Comment:
Is the mask value used? If not, does this type need to be generic over it? (I assume yes, for the sake of calling code?)

Reply:
At least for now, there is no metadata that embeds payload within a function pointer, so we actually don't use the mask value